### PR TITLE
Feature/pdf2

### PR DESCRIPTION
**Pull request by LennyDZ**
Add pdf picker to add_deadline_activity

### CODE REVIEW

**Review by Matathiasq13824 — CHANGES_REQUESTED** [review]
Nothing more to add to my comments. The coverage is a little low but tests can be added later on in the next sprint

File: app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt [hunk 1]
```diff
@@ -208,6 +219,21 @@ class AddDeadlineActivity : AppCompatActivity() {
             }
         }
     }
+    private val startForResult =
```
Comment (Matathiasq13824) [inline app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt:222]: As we search a pdf, could this be tested ?

File: app/src/main/res/layout/activity_add_deadline.xml [hunk 2]
```diff
@@ -158,8 +194,11 @@
...
             app:layout_constraintHorizontal_bias="0.502"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/constraintLayout2"
-            tools:ignore="TextContrastCheck" />
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.72" />
+
+        app:layout_constraintTop_toBottomOf="@+id/constraintLayout2"
+        tools:ignore="TextContrastCheck" />
     </androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (Matathiasq13824) [inline app/src/main/res/layout/activity_add_deadline.xml:202]: You have a warning here. You should correct it

File: app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt (hunk 1 above)
Comment (LennyDZ) [reply app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt:222]: I tried. Haven't found yet how to do it

**Review by Matathiasq13824 — APPROVED** [review]

**Review by Voxanimus — APPROVED** [review]
I don't know what you exactly did on your last PR but the modification i saw sounds reasoable

File: app/src/main/java/com/github/multimatum_team/multimatum/util/PDFUtil.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import android.database.Cursor
+import android.net.Uri
+import android.provider.OpenableColumns
+
+
+object PDFUtil {
+    /**
+     * Intent for navigating to the files to select PDF
+     */
+    fun selectPdfIntent(callback: (Intent) -> Unit) {
```
Comment (felko) [inline app/src/main/java/com/github/multimatum_team/multimatum/util/PDFUtil.kt:13]: Why did you need to create this object instead of putting the function in the `AddDeadlineActivity.selectPDF` method?

**Review by felko — APPROVED** [review]
I'm approving I'm not here to approve later tonight or early tomorrow, but please fix the issue before merging if you agree
